packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -91,20 +91,23 @@ class MiniCart extends Component {
         const { cart, classes } = this.props;
         const { cartCurrencyCode, cartId } = this;
         const hasSubtotal = cartId && cart.totals && 'subtotal' in cart.totals;
+        const itemsQuantity = cart.details.items_qty;
+        const itemQuantityText = itemsQuantity === 1 ? 'item' : 'items';
+        const totalPrice = cart.totals.subtotal;
 
         return hasSubtotal ? (
             <dl className={classes.totals}>
                 <dt className={classes.subtotalLabel}>
                     <span>
-                        Subtotal
-                        {` (${cart.details.items_qty} Items)`}
+                        Cart Total :&nbsp;
+                        <Price
+                            currencyCode={cartCurrencyCode}
+                            value={totalPrice}
+                        />
                     </span>
                 </dt>
                 <dd className={classes.subtotalValue}>
-                    <Price
-                        currencyCode={cartCurrencyCode}
-                        value={cart.totals.subtotal}
-                    />
+                    &nbsp;({itemsQuantity} {itemQuantityText})
```

Comment:
Generally, we shouldn't be adding spacing in the content. We should try to render these nodes such that they're perfectly adjacent and then space them with CSS.